### PR TITLE
zh, en: add default value for --flush flag for v2.0.5

### PR DESCRIPTION
**Pull request by GMHDBJD**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/manage-schema.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ Global Flags:
 + `--flush`:
     - 可选
     - 同时将表结构写入 checkpoint，从而在任务重启后加载到该表结构
+    - 默认值为 `true`
```
Comment (lance6716) [inline zh/manage-schema.md:88]: 这样写会让我以为是 `--flush true` 这么用。应该是 2.0.5 默认启用？

Comment (GMHDBJD) [reply zh/manage-schema.md:88]: 目的是让用户不想用的话，--flush false。。。

**Review by lance6716 — APPROVED** [review]

**Review by Liuxiaozhen12 — APPROVED** [review]

**Pull request merged**